index.html

```diff
@@ -1,21 +1,67 @@
-<!--So, in addition to building an application that calculates daily sales projections for each location (in a file called sales.html), you also need to create a pubic-facing page (in a file called index.html) that is colorful, eye-catching, readable, useful, informative... and ultimately of a quality ready for use on T-shirts, stickers, coffee mugs, cookie bags, napkins, and so on.-->
-
 <!DOCTYPE html>
+
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title></title>
-  </head>
-  <body>
+<head>
+	<meta charset="utf-8">
+	<link href="style.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Anton|Source+Sans+Pro" rel="stylesheet">
+	<script src="https://use.fontawesome.com/20877e6602.js">
+	</script>
 
-  </body>
-</html>
+	<title>
+	</title>
+</head>
+
+<body id="body">
+	<img alt="Photo of a Salmon" id="salmon" src="img/salmon.png">
+
+	<h1>Pat's Salmon Cookies</h1>
+
+
+	<h2><i aria-hidden="true" class="fa fa-quote-left"></i> Pat's Salmon Cookies are the hands down best in the world!! <i aria-hidden="true" class="fa fa-quote-right"></i><br>
+	--Anthony Bourdain</h2>
+
+	<hr>
+
+
+	<p>Welome to our small famy biz! Our small family loves fish. One day our kid Pat found a fish cookie cutter. From there we decided to make some really gross cookies. At first they were pretty rough looking. The cookies slowly started to look better and biz is booming. So, like every American store we also sell t-shirt swag.</p>
+
+
+	<div class="slideshow">
+		<div class="slide current"><img alt="" src="img/family.jpg">
+		</div>
 
 
-// renderHTML: function() {
-//   var newUl = document.createElement('ul');
-//   var newLi = document.createElement('li');
-//   newLi.innerText = storeHours[0] + ': ' + this.salesArray[0] + 'cookies';
-//   newUl.appendChild(newLi);
-//   var body = document.getElementById('body');
-//   body.appendChild(Ul);
+		<div class="slide"><img alt="" src="img/chinook.jpg">
+		</div>
+
+
+		<div class="slide"><img alt="" src="img/cutter.jpeg">
+		</div>
+
+
+		<div class="slide"><img alt="" src="img/frosted-cookie.jpg">
+		</div>
+
+
+		<div class="slide"><img alt="" src="img/fish.jpg">
+		</div>
+
+
+		<div class="slide"><img alt="" src="img/shirt.jpg">
+		</div>
+	</div>
+	<span id="previous-button">Previous</span> <span id="next-button">Next</span>
+
+	<hr>
+
+
+	<footer>
+		Hours: 9am - 5pm, M-F<br>
+		Location: 1234 Pike Street, Seattle, WA 98118<br>
```

Comment:
You don't need to use line breaks (though they should be self-closing when you use them). Appropriately-styled `<p>` tags should work just fine.